src/rgw/motr/gc/gc.cc

```diff
@@ -14,16 +14,60 @@
  */
 
 #include "motr/gc/gc.h"
+#include <cstdlib>
+#include <ctime>
 
 void *MotrGC::GCWorker::entry() {
   std::unique_lock<std::mutex> lk(lock);
   ldpp_dout(dpp, 10) << __func__ << ": " << gc_thread_prefix
     << worker_id << " started." << dendl;
 
+  // Get random number to lock the GC index.
+  uint32_t my_index = rand() % (motr_gc->max_indices);
```

Comment:
This without a seeder will generate the same sequence of numbers every time it is called. 
So either we can use `srand()` or we can use CEPH's random number function `ceph::util::generate_random_number()`

Reply:
Done